### ISSUE TITLE
TODO about further tests for IRETURN or PARAMVALUE in _imethodcall()

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#919]
This issue is to resolve the following TODOs in `_imethodcall()` in cim_operations.py:
```
W:1342, 0: TODO Further tests on this IRETURN or PARAMVALUE (fixme)
```

This needs some investigation first.

**Comment by andy-maier** [comment]
In general, these elements can be tested at these two levels:
- In the tupleparser. The IRETURNVALUE, IPARAMVALUE, RETURNVALUE, PARAMVALUE elements are not covered there at this point, and this level would be used to test the possible valid and invalid XML strings exhaustively. However, that does not allow finding out whether the child element of these elements matches what the respective operation requires.
- In test_client yaml files. These elemens are already tested there for all operations, but the coverage of possible input parameters is far from complete.

It seems that completing the tests at the test-client yaml level would be appropriate for this issue.

**Comment by andy-maier** [comment]
A pragmatic approach for addressing this issue would be to go throughthe testclient yaml files and to add TODOs there, for any coverage holes we see.

**Comment by KSchopmeyer** [comment]
We proposed to give Tony until after 4th of July to clean up his PR and then take it over.

**Comment by andy-maier** [comment]
This issue is addressed by PR #1322
